lib/patriot/job_store/rdb_job_store.rb

```diff
@@ -401,7 +401,7 @@ def _set_dependency(db_conn, direction, depth, nodes, edges, base_job)
       def find_jobs_by_state(state, opts = {})
         raise "OFFSET is set WITHOUT LIMIT" if opts.has_key?(:offset) && !opts.has_key?(:limit)
         condition = ["state = #{state}", "id != #{@initiator_id}"]
-        condition |= ["job_id LIKE '#{opts[:filter_exp]}'"] if opts.has_key?(:filter_exp)
+        condition |= ["job_id LIKE '%#{opts[:filter_exp]}%'"] if opts.has_key?(:filter_exp)
```

Comment:
Thank you  for the patch. Adding '%' to the head of `filter_exp` would make the index of `job_histories` table ineffective. Would you modify the patch to add '%' only to the tail of `filter_exp`?